databricks-sdk-java/src/test/java/com/databricks/sdk/integration/CredentialsIT.java

```diff
@@ -15,10 +15,13 @@
 public class CredentialsIT {
   @Test
   void lists(AccountClient a) {
-    Iterable<Credential> list = a.credentials().list();
+    // Skipping this test for gcp because there is a permission issue with listing credentials.
```

Comment:
Why is this?

Reply:
Updated the comment, as discussed offline, this API is not enabled for GCP, so skipping it